src/adapters/moonbeam.ts

```diff
@@ -1,13 +1,28 @@
-import { FixedPointNumber as FN } from "@acala-network/sdk-core";
-import { Observable } from "rxjs";
-
+import { Storage } from "@acala-network/sdk/utils/storage";
+import { AnyApi, FixedPointNumber as FN } from "@acala-network/sdk-core";
+import { combineLatest, from, map, Observable } from "rxjs";
+import { DeriveBalancesAll } from "@polkadot/api-derive/balances/types";
 import { SubmittableExtrinsic } from "@polkadot/api/types";
-import { ISubmittableResult } from "@polkadot/types/types";
-
 import { BaseCrossChainAdapter } from "../base-chain-adapter";
 import { ChainName, chains } from "../configs";
-import { ApiNotFound } from "../errors";
-import { BalanceData, BasicToken, CrossChainTransferParams } from "../types";
+import { ApiNotFound, CurrencyNotFound } from "../errors";
+import { BalanceData, BasicToken, CrossChainRouterConfigs, CrossChainTransferParams } from "../types";
+import { BalanceAdapter, BalanceAdapterConfigs } from "../balance-adapter";
+import { ISubmittableResult } from "@polkadot/types/types";
+import { BN } from "@polkadot/util";
+
+const DEST_WEIGHT = "800000000";
+
+export const moonriverRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
+  {
+    to: "calamari",
+    token: "MOVR",
+    xcm: {
+      fee: { token: "MOVR", amount: "2000000000000000" },
+      weightLimit: DEST_WEIGHT,
```

Comment:
weight limit of MOVR from moonriver to calamari is `1006050000`, so this weight limit(800000000) is not enough. also fee is `503025000000000`.

Reply:
I will change the weight limit to be "Unlimited" if you think that's okay. It seems like all the wait limits throughout this repo are set to be either too high or "Unlimited". I think "Unlimited" is now preferred, as Acala's adapter was recently update to use "Unlimited":
https://github.com/polkawallet-io/bridge/pull/17/files

My fee calculation is:
```
UNITS_PER_SECOND = 500,000,000,000,000,000 (queried through asset manager)
WEIGHT_PER_SECOND = 1,000,000,000,000 (runtime constant)
WEIGHT = 4 instructions * 1,000,000,000 weight per instruction = 4,000,000,000
DESTINATION_FEE = UNITS_PER_SECOND * WEIGHT / WEIGHT_PER_SECOND = 2,000,000,000,000,000
```
Can you let me know how you are getting 1006050000?